packages/app/src/specs/SpecsList.vue

```diff
@@ -206,11 +207,13 @@ const cachedSpecs = useCachedSpecs(
   computed(() => props.gql.currentProject?.specs ?? []),
 )
 
+const specsListHeader = ref('')
 const search = ref('')
 const debouncedSearchString = useDebounce(search, 200)
 
 function handleClear () {
   search.value = ''
+  specsListHeader.value.$refs.customInput.$refs.input.focus()
```

Comment:
this seems a bit confusing...is it easier/preferred to just do `document.getElementById('spec-filter').focus()`?

not sure what the best practice/convention is here

Reply:
We can do the "forward ref" pattern (same as React, if you've seen it there before).

I pushed a commit: https://github.com/cypress-io/cypress/pull/21315/commits/ccd48a95f9c11300d7880a36828847d12cae4577.

It's basically what you were doing, but instead of deeply reaching down via `$refs`, we just pass a `ref` as a prop. To pass it, you need to wrap it in a closure. You can read more about that here: https://github.com/vuejs/composition-api/issues/317.